doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/managing-iot-data.md

```diff
@@ -1 +1,558 @@
-# To be replaced with content
+# Managing IoT data
+With the TourGuide-App, you can generate, simulate and give persistence to sensors measurements.  For that purpose, we use the information of restaurants, and
+generate humidity and temperature sensors for the kitchen and dining room of a restaurant.  Every measurement and information regarding the sensors will be finally propagated to Cygnus, using MySQL database by default.
+
+Note that, to this aim, we will need to previously load the Restaurants information. For further information on how to generate them just execute the load command with the `--help` parameter:
+
+```
+./tour-guide load --help
+```
+
+And make sure the required services (`tourguide, orion, idas and cygnus`) are up and running.
+
+Once done, we can start working with sensors.
+
+## Working with sensors
+
+In TourGuide-App we provide different ways to work with sensors:
+
+```
+$ ./tour-guide sensors
+Usage: tour-guide sensors [-h | --help] <command> <options>
+
+Run sensors related commands:
+
+  create                 	Create sensors for the restaurants available in the application.
+  update                 	Update all restaurant sensors measurements.
+  send-data              	Send a single measurement for a specific sensor.
+  simulate-data          	Simulate a sensor sending data over a period of time.
+
+Command options:
+
+  -h  --help             	Show this help.
+
+Use 'tour-guide sensors <command> --help' to get help about a specific <command>.
+```
+
+## Creating sensors
+
+Creating sensors is as simple as running:
+
+```
+./tour-guide sensors create
+```
+
+This will create and initialize four sensors for each of the available restaurants in the application. There will be temperature and relative humidity sensors for both the kitchen and the dining room of the restaurant.  This command executes a script ([sensorsgenerator.js](https://github.com/Fiware/tutorials.TourGuide-App/blob/develop/server/feeders/sensorsgenerator.js)) that uses the IoT agent API to create the sensors and initialize them with a default measurement.
+
+If you want to create the sensors yourself, you can follow this steps:
+
+### Register a new service with the IoT Agent
+
+The first step is to register a new service configuration with the IoT Agent (if it doesn't exists).  To do this we need to send a POST request to `http://localhost:4041/iot/services` with the following JSON payload:
+```
```

Comment:
Port 4041 is exposedin the host, so it's not needed to call `idas:4041`. To call `idas:4041` we need first to configure the hostfile by doing: 

```bash
./tour-guide configure hosts -m
```